src/libraries/Common/src/Interop/Android/System.Security.Cryptography.Native.Android/Interop.X509Store.cs

```diff
@@ -57,6 +57,12 @@ internal static partial class AndroidCrypto
             SafeX509StoreHandle store,
             SafeX509Handle cert,
             string hashString);
+
+        [LibraryImport(Libraries.AndroidCryptoNative, EntryPoint = "AndroidCryptoNative_X509StoreGetPrivateKeyEntry", StringMarshalling = StringMarshalling.Utf8)]
+        internal static partial IntPtr X509StoreGetPrivateKeyEntry(IntPtr store, string hashString);
+        [LibraryImport(Libraries.AndroidCryptoNative, EntryPoint = "AndroidCryptoNative_X509StoreDeleteEntry", StringMarshalling = StringMarshalling.Utf8)]
+        [return: MarshalAs(UnmanagedType.Bool)]
```

Comment:
Is this `MarshalAs(...Bool)` right? Up above we have it as `UnmanagedType.U1`. Looking at the native definition, this seems "off" to me because a C bool is usually 1 byte whereas `UnmanagedType.Bool` is a 4 byte value.